tests/v2/validation/snapshot/snapshot_additional_test.go

```diff
@@ -127,6 +127,12 @@ func (s *SnapshotAdditionalTestsTestSuite) TestSnapshotRecurringRestores() {
 		err = v1.ConvertToK8sType(cluster, &updatedCluster)
 		require.NoError(s.T(), err)
 
+		if updatedCluster.Spec.RKEConfig.ETCD.S3 != nil {
+			tt.name = "S3 " + tt.name
+		} else {
+			tt.name = "Local " + tt.name
```

Comment:
hard to tell from this, will wait on jenkins job to confirm but concerned this will read as 'local cluster' vs. local snapshot

Reply:
The test case will be `TestSnapshotRestoreTestSuite/TestSnapshotRestoreETCDOnly/RKE2_Local_Restore_etcd_only`.

Reply:
Let me push a commit, I have it switched by accident.